components/category-toolgroup.vue

```diff
@@ -98,11 +99,10 @@
                     while (labelIndex < label.length) {
                         if (label[labelIndex].toLowerCase() === searchChar) {
                             matchFound = true;
-                            labelIndex++;
                             break;
                         }
-                        labelIndex++;
                     }
+                    labelIndex++;
```

Comment:
has to be above the `if` to work correctly!

Reply:
Now it makes sense why the e2e tests have been hanging 😅

Reply:
I reckon we're skipping the 2nd character now